packages/fast-colors/src/colorlib/colorPalette.ts

```diff
@@ -0,0 +1,271 @@
+// tslint:disable:member-ordering
+// tslint:disable:prefer-for-of
+// tslint:disable:interface-name
+
+import { blendMultiply, blendOverlay, saturateViaLCH } from "./colorBlending";
+import { rgbToHSL } from "./colorConverters";
+import { ColorHSL } from "./colorHSL";
+import { ColorInterpolationSpace, interpolateByColorSpace } from "./colorInterpolation";
+import { ColorRGBA64 } from "./colorRGBA64";
+import { ColorScale } from "./colorScale";
+
+export interface IColorPaletteConfig {

```

Comment:
nit: can we remove the "I" prefix? It's suggested by typescript that interfaces do not start with "I" as the name of the interface should be axiomatic (and appears to be in this case).